addons/website_hr_recruitment/static/src/scss/website_hr_recruitment.scss

```diff
@@ -1,24 +1,89 @@
-.o_website_hr_recruitment_jobs_list {
-    #jobs_grid_left {
-        ul.flex-column > li.nav-item > a {
-            padding: .5rem 1rem;
-            border-radius: 0.125rem;
+@mixin o_jobs_has_shadows {
+    &.o_jobs_has_shadows {
+        &, * {
+            transition: all .3s;
         }
-    }
-    .card {
-        padding: 15px;
-        box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.17);
-
         &:hover {
-            transition: box-shadow 0.3s cubic-bezier(0.55, 0, 0.1, 1);
             box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.1), 0 2px 2px 0 rgba(0, 0, 0, 0.05);
         }
+    }
+}
+@mixin o_jobs_has_color {
+    &.o_jobs_has_color {
```

Comment:
Not sure about this design :/
A card that gets a primary coloured border on hover only... it's not be consistent with all the other modules that use either cards or lists.